test/functional/viml/max_args_spec.lua

```diff
@@ -0,0 +1,29 @@
+local helpers = require('test.functional.helpers')
+
+local clear = helpers.clear
+local eq = helpers.eq
+local exc_exec = helpers.exc_exec
+
+describe('Up to MAX_FUNC_ARGS arguments are handled by', function()
+  local max_func_args = 20  -- from eval.h
```

Comment:
I'm doing that already, great. :-)

BTW, I could also add a test for `rpcrequest()`, but it would be the same as for `rpcnotify()`, hence I skipped it.